### PR TITLE
Lab meeting blog | 3 October 2019

### PR DESCRIPTION
**Pull request by KirstieJane**
Not too much in this one - hopefully quick to review

### CODE REVIEW

**Comment by KirstieJane** [comment]
@minnieho1115 - I didn't include your slides from this blog post because I wasn't clear on whether they were for public sharing. They could also go in #90 if they are ok to be public?

File: _posts/blog/2019-10-03-2019-10-03-lab-meeting.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+modified:
+share: true
+author: kirstie_whitaker ### CHANGE to your author name (in _data/authors.yml)
+---
+
+There was no lab meeting this week because there was a Turing Town Hall (where we get to grill the [Institute Director](https://www.turing.ac.uk/people/leadership/adrian-smith)).
+
+We are delighted to share this video from May of the [Pint of Science](https://www.turing.ac.uk/events/pint-science) panel discussion featuring Kirstie Whitaker, [Dianna Smith](https://twitter.com/geodianna?lang=en), [Rebecca Sterry](https://twitter.com/bex_sterry), [Suzanne Iwai](https://twitter.com/SuzanneIwai) and Georgia Aitkenhead on **Empowering autistic people through citizen science**.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/DNisoU9uXqk" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (sgibson91) [inline _posts/blog/2019-10-03-2019-10-03-lab-meeting.md:20]: This video doesn't render in markdown (but it might not need to?)

**Review by sgibson91 — APPROVED** [review]
I left one comment which may not be relevant depending on how clever Jekyll is 😄Leaving an approving review anyway.

**Review by LouiseABowler — APPROVED** [review]

File: _posts/blog/2019-10-03-2019-10-03-lab-meeting.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+* Slates: [https://tinyurl.com/y37wk4zh](https://tinyurl.com/y37wk4zh)
+* Voting: [at this google form](https://docs.google.com/forms/d/e/1FAIpQLSfG76N3dnazONsfCsWAXccgrizmb1gm233pJSow3TmTUi3_Fw/viewform) (the URL is too long 😆)
+
+**Patricia** came across this course about ways of thinking in informatics and thinks it should be mandatory for everyone working with data! [https://wot.pubpub.org](https://wot.pubpub.org) 💯
+She also recommends watching [Angela Saini](https://www.angelasaini.co.uk) on BBC in ["Eugenics: Science's Greatest Scandal"](https://www.bbc.co.uk/programmes/m0008zc5)
+
+**Konrad** has enjoyed working with Kirstie, Petra Vertes and Ross Markello this week to pythonise their analyses of [Allen Human Brain Atlas](https://human.brain-map.org/) gene data.
+They were working on making sure that Ross' pyls package: [https://github.com/rmarkello/pyls](https://github.com/rmarkello/pyls) (which unfortunately needs to be renamed because someone already [has that name on PyPI](https://pypi.org/project/pyls) 😢).
+
+**Yo** has her PhD programme's 9-month review next week to double check if I'm good enough to continue researching with Manchester.
```
Comment (yochannah) [inline _posts/blog/2019-10-03-2019-10-03-lab-meeting.md]:
```suggestion
**Yo** has her PhD programme's 9-month review next week to double check if she's good enough to continue researching with Manchester.
```

**Review by yochannah — APPROVED** [review]

**Review by Islast — APPROVED** [review]
This is great stuff. I missed this one (whoops) and it's really fun reading back over all the cool stuf people got up to. That Allen human brain atlas stuff sounds very exciting :+1:

File: _posts/blog/2019-10-03-2019-10-03-lab-meeting.md (hunk 1 above)
Comment (KirstieJane) [reply _posts/blog/2019-10-03-2019-10-03-lab-meeting.md:20]: Thanks @sgibson91 - the video _does_ render in jekyll but I appreciate you checking ✨

**Review by minnieho1115 — APPROVED** [review]

**Pull request merged**